ChangeLog.d/9302.txt

```diff
@@ -0,0 +1,6 @@
+Features
+   * Added new configuration option MBEDTLS_PSA_STATIC_KEY_SLOTS, which
+     uses static storage for keys, enabling malloc-less use of key slots.
+     The size of each buffer is given by the option
+     MBEDTLS_PSA_STATIC_KEY_SLOT_BUFFER_SIZE. By default it accomodates the
```

Comment:
(spelling) sorry, missed it before, but this should be `accommodates`.

Other than that, this all looks good to me